tasks/eslint.md

```diff
@@ -0,0 +1,104 @@
+10. [brace-style](http://eslint.org/docs/rules/brace-style) (7 errors) (--fix) 
+    to be changed to `['error', '1tbs', { allowSingleLine: true }]`
+    ([airbnb](https://github.com/airbnb/javascript#blocks--cuddled-elses))
+11. [keyword-spacing](http://eslint.org/docs/rules/keyword-spacing) (12 errors) (--fix)
+    ([airbnb](https://github.com/airbnb/javascript#whitespace--around-keywords)) to be changed to 
+
+    ```
+       'keyword-spacing': ['error', {
+          before: true,
+          after: true,
+          overrides: {
+            return: { after: true },
+            throw: { after: true },
+            case: { after: true }
+          }
+        }],
+    ```
+
+12. [one-var](http://eslint.org/docs/rules/one-var) (32 errors) to be changed to `['error', 'never']` 
+13. [no-else-return](http://eslint.org/docs/rules/no-else-return) (39 errors)
+14. [no-plusplus](http://eslint.org/docs/rules/no-plusplus) (40 errors)
+    ([airbnb](https://github.com/airbnb/javascript#variables--unary-increment-decrement))
+15. [no-shadow](http://eslint.org/docs/rules/no-shadow) (56 errors)
+16. [max-len](http://eslint.org/docs/rules/max-len) (78 errors)
+    ([airbnb](https://github.com/airbnb/javascript#whitespace--max-len)) to be changed to 
+
+    ```
+        'max-len': ['error', 100, 2, {
+          ignoreUrls: true,
+          ignoreComments: false,
+          ignoreRegExpLiterals: true,
+          ignoreStrings: true,
+          ignoreTemplateLiterals: true,
+        }],
+    ```
+
+17. [quote-props](http://eslint.org/docs/rules/quote-props) (201 errors) (--fix) to be changed
+    to `['error', 'as-needed', { keywords: false, unnecessary: true, numbers: false }]`
+    ([airbnb](https://github.com/airbnb/javascript#objects--quoted-props))
+18. [no-unused-vars](http://eslint.org/docs/rules/no-unused-vars) (221 errors) to be changed
+    to `['error', { vars: 'local', args: 'after-used' }]` 
+19. [no-underscore-dangle](http://eslint.org/docs/rules/no-underscore-dangle) (410 errors)
+    ([airbnb](https://github.com/airbnb/javascript#naming--leading-underscore))
+20. [comma-dangle](http://eslint.org/docs/rules/comma-dangle) (795 errors) (--fix)
+    ([airbnb](https://github.com/airbnb/javascript#commas--dangling)) to be changed to 
+
+    ```
+        'comma-dangle': ['error', {
+          arrays: 'always-multiline',
+          objects: 'always-multiline',
+          imports: 'always-multiline',
+          exports: 'always-multiline',
+          functions: 'always-multiline',
+        }],
+    ```
+
+21. [camelcase](http://eslint.org/docs/rules/camelcase) (3680 errors)
+    ([airbnb](https://github.com/airbnb/javascript#naming--camelCase)) to be changed 
+    to `['error', { properties: 'never' }]` 
+22. [quotes](http://eslint.org/docs/rules/quotes) (5773 errors) (--fix) to be changed to 
+    `['error', 'single', { avoidEscape: true }]`
+    ([airbnb](https://github.com/airbnb/javascript#strings--quotes))
```

Comment:
These last two are probably too big to be GCI tasks; refactoring 1000s of things is a ton of work, and I'm also not confident we want to change the first one.